develop/components/pages/SliderPage.tsx

```diff
@@ -32,9 +32,14 @@ const SliderExamples: React.FunctionComponent = () => {
     const [disabledSlider1, setDisabledSlider1] = React.useState<number>(2500);
     const [sebSlider, setSebSlider] = React.useState<number>(0);
     const [sebSliderError, setSebSliderError] = React.useState<string>("");
+    const [diverse, setDiverse] = React.useState<number>(diverseLabels[0].position);
```

Comment:
check if diverseLabels[0] exist before checking the position propety, otherwise it will throw "cannot find position of undefined "

Reply:
This is the `sliderPage` file, and `diverseLabels` is a predefined constant array. It will always exist because it's part of the documentation